docs/sources/whatsnew/whats-new-in-v7-5.md

```diff
@@ -0,0 +1,103 @@
++++
+title = "What's new in Grafana v7.5"
+description = "Feature and improvement highlights for Grafana v7.5"
+keywords = ["grafana", "new", "documentation", "7.5", "release notes"]
+weight = -32
+[_build]
+list = false
++++
+
+# What’s new in Grafana v7.5
+
+> **Note:** This is content for a beta version of Grafana. It will be updated several times before the stable release.
+
+This topic includes the release notes for Grafana v7.5. For all details, read the full [CHANGELOG.md](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
+
+## Grafana OSS features
+
+These features are included in the Grafana open source edition.
+
+### Pie chart panel visualization (beta)
+
+Grafana 7.5 adds a beta version of the next-gen pie chart panel.
+
+### Alerting for Loki
+
+Grafana 7.5 comes with alerting support for Loki. With LogQL you can wrap a log query with the functions that allow for creating metrics out of the logs, such as "rate()". Metric queries can then be used to calculate things such as the rate of error messages. [When combined with log parsers](https://www.youtube.com/watch?v=H9z2V0Ib1q0), they can be used to calculate metrics from a value within the log line, such latency or request size.
+
+With alerting support for Loki, you can now create alerts on Loki metrics queries. Alert rules can be set up easily in the graph panel and you can learn more about the process of creating alerts in the Alert documentation.
+
+![Loki alerting](/img/docs/alerting/alerting-for-loki-7-5.png)
+
+### Loki label browser
+
+A new Loki logs browser lets you construct the queries step by step: you choose labels that you like to consider, such as "job", then you select the values that you like, such as "my-app1". Note that you can select values from more than one label, and that they get facetted. This means only possible label combinations are selectable. When you're done, you can run the query either as a logs or a metrics query (a metrics query returns the log volume in a chart).
+
+### Changed default HTTP method for new Prometheus data sources
+
+For new Prometheus data sources, we have changed the default HTTP method to POST. POST allows for much larger query bodies than using the GET method. This is necessary when sending queries from graphs with a lot of targets, such as many hosts in a dashboard variable. POST also makes the Query Inspector data easier to read since the query is in plain text where the GET query is URL encoded.
```

Comment:
```suggestion
For new Prometheus data sources, we have changed the default HTTP method to POST. POST allows for much larger query bodies than using the GET method. This is necessary when sending queries from graphs with a lot of targets, for example, many hosts in a dashboard variable. The POST method also makes the Query Inspector data easier to read since the query is in plain text whereas the GET query is URL encoded.
```